CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# v1.23.0 (2024-09-19)
+
+## Orchestrator Changes
+
+### Kubernetes
+* Support Kubernetes NVIDIA Device Plugin configurations through API ([#4182])
+* Support NVIDIA Container Toolkit configurations through API ([#4182])
+
+## Build Changes
+* Update bottlerocket-sdk to 0.45 ([#4189])
+
+### Twoliter
+* Update bottlerocket-core-kit to v2.5.0 ([#4189])
+* Perform binary checksum validation ([#4192])
+* Update Twoliter to v0.4.6 ([#4200])
+* Update bottlerocket-core-kit to v2.6.0 ([#4203])
```

Comment:
Should we merge both Core kit releases and say "Update bottlerocket-core-kit to v2.6.0 ([#4189], [#4203])"

Reply:
I guess I can do that without calling out the version

Reply:
While updating the website I check the changelog to get the core-kit version. I would prefer the mention of the version in one of the lines.

Reply:
Reference: https://github.com/bottlerocket-os/bottlerocket/pull/4157